src/main/java/com/hibernate/model/Movie.java

```diff
@@ -0,0 +1,39 @@
+package com.hibernate.model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Movie {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private String title;
+    private String description;
+
+    public long getId() {
```

Comment:
```suggestion
    public Long getId() {
```

Reply:
how about this comment?

Reply:
![image](https://user-images.githubusercontent.com/75170409/105988987-23b90680-60a9-11eb-817d-d08dac1652d9.png)